src/blocks/recommendations/block.json

```diff
@@ -0,0 +1,44 @@
+{
+	"$schema": "https://schemas.wp.org/wp/5.9/block.json",
+	"apiVersion": 2,
+	"name": "wp-parsely/recommendations",
+	"title": "Parse.ly Recommendations",
+	"category": "widgets",
+	"textdomain": "wp-parsely",
+	"supports": {
+		"html": false
+	},
+	"attributes": {
+		"boost": {
+			"type":"string",
+			"default": "views"
+		},
+		"imagestyle": {
+			"type": "string",
+			"default": "original"
+		},
+		"limit": {
+			"type": "number",
+			"default": 3
+		},
+		"showimages": {
+			"type": "boolean",
+			"default": true
+		},
+		"sort": {
+			"type":"string",
+			"default":"score"
+		},
+		"tag": {
+			"type":"string"
+		},
+		"title": {
+			"type":"string",
+			"default": "Related Content"
+		}
+	},
+	"editorScript": "file:./edit.js",
+	"style": "file:./style-edit.css",
+	"editorStyle": "file:./edit.css",
+	"viewScript": "file:./view.js"
```

Comment:
Noting that using / relying on the block.json means WP 5.8 is the minimum, and `viewScript` was only supported in WP 5.9.

Reply:
I was trying to remember why I didn't use the json for this in the first place and the minimum WordPress version is it.

Reply:
What is the minimum version this block needs to support?  I probably should have asked that first 😄

Reply:
OK, so the Block in its initial state was supporting WP 5.6 (released December 2020). [That line is still present in class-recommendations-block.php](https://github.com/Parsely/wp-parsely/blob/ed18932ef133a4dd3d8f28492084e43d44e90a78/src/blocks/recommendations/class-recommendations-block.php#L19). Personally, I don’t see a need to support such an old version.

The big change with this PR is that _the block now works in FSE_, which wasn’t the case before. I clearly see the point that supporting 5.9 and upwards is only supporting the “bleeding edge”. My suggestion is that if we can support 5.8 or 5.7 with very minor changes that will be easy to update in a few months, we can go that route. _This provided of course that the block will keep working in FSE_.

Else I’m totally fine with supporting 5.9 and upwards, as users who care for their WP installations will sooner or later get there. WordPress 6.0 release is in 24 May and we generally encourage people to upgrade, so 5.9 will not be that new in a couple of months.

Reply:
We have released the block on version 3.2.0 of the plugin under a feature flag, which gives us the flexibility of doing changes as such. I think Alex is making a very good point about the potential users of the block. 

I am going to argue that the full benefits of the block come with 5.9, so I don't want to commit now to support 5.8, which will probably bring us difficulties down the line. If we all agree, I would strongly push for supporting the block just on 5.9+ and having the least amount of "hacks".

Reply:
I do agree with the idea that there should be a "decision factor" besides development effort (though effort is definitively a big thing for us as a small team). In cases like this one, I think it should be based on data and goals to help us make correct decisions.

For example, with some statistics on the WP versions used by our users, and (let's say) an intention to keep compatibility with at least 70% of them, this could orient us. Without such rules and data, making correct decisions is harder. Trying to cover almost every user is always a great idea, but it can be costly in practice and limited in returns.

If backwards compatibility becomes problematic for us or for users that do update, I would go for serving the users who update. If we abort the idea of `block.json`, we'll have to find a solution for users who want to have FSE available. But keeping 5.8 compatibility is simple.

All this to say that there is a point in all arguments made here and that we could take this discussion further if needed.